### DISCUSSION TITLE
Is it possible to make database avaible in the icon menu?

### DISCUSSION THREAD

**Discussion by Yacson** [keepassxreboot/keepassxc#10072] [Ideas]
Hi,

To not have aways the app open on my mac, I was thinking if it will be possible to work on some kind of floating panel like feature (https://github.com/mitchcapper/KPFloatingPanel). 

Here, the idea is to incorporate this functionality to KeePassXC icon in the menu bar. I am not really a developer but I am willing to help.

![2023-12-02_09-42-05](https://github.com/keepassxreboot/keepassxc/assets/7979717/e5c06869-8b12-465d-a8cd-90514b2dfd1e)

Thanks for your consideration :-)

**Comment by Yacson** [comment]
How to promote ideas here?

**Reply by droidmonkey** [reply]
Hello, I saw this but it's a massive undertaking to do what you are asking and very unlikely to happen. We offer a lot of UX advantages over KeePass original that make this feature unnecessary.

**Reply by Yacson** [reply]
I won't say it is unnecessary but for me it is always about the user experience.
This ico is already there so all we need to add additional feature. The user will not always where is keepassXC. and you open the app only when it comes to modify or create a new key. 
it make it nicer and less invasive
